src/gui/qgsadvanceddigitizingdockwidget.h

```diff
@@ -399,6 +472,9 @@ class GUI_EXPORT QgsAdvancedDigitizingDockWidget : public QgsDockWidget, private
      */
     void activateCad( bool enabled );
 
+    //! enable/disable floater
+    void toggleFloater( bool enabled );
```

Comment:
I'd call that `setFloaterEnabled`, toggle sounds like it will turn if on and off if called repeatedly without paramter.

Reply:
removed this method (small refactoring mentionned above)